src/mono/mono/metadata/custom-attrs.c

```diff
@@ -2488,8 +2488,8 @@ mono_reflection_get_custom_attrs_info_checked (MonoObjectHandle obj, MonoError *
 		cinfo = mono_custom_attrs_from_builders_handle (NULL, &dynamic_image->image, cattrs);
 	} else if (strcmp ("MonoGenericClass", klass_name) == 0) {
 		MonoReflectionGenericClassHandle gclass = MONO_HANDLE_CAST (MonoReflectionGenericClass, obj);
-		MonoReflectionTypeHandle generic_type = MONO_HANDLE_NEW_GET (MonoReflectionType, gclass, generic_type);
-		cinfo = mono_reflection_get_custom_attrs_info_checked (MONO_HANDLE_CAST (MonoObject, generic_type), error);
+		MonoReflectionTypeHandle _genericType = MONO_HANDLE_NEW_GET (MonoReflectionType, gclass, _genericType);
```

Comment:
It looks weird to use `_` prefix for local variables. It does not follow any of the coding conventions used in the repo. We would not do it in managed code either.

Also, it is not necessary for the unmanaged field names and the managed field names for match exactly. One example from many:
- Mono unmanaged field names: https://github.com/dotnet/runtime/blob/6fecc25b0260b1586305ecefe0cef9a0e22fb66c/src/mono/mono/metadata/class-internals.h#L684-L685
- Managed field names: https://github.com/dotnet/runtime/blob/main/src/libraries/System.Private.CoreLib/src/System/Runtime/InteropServices/HandleRef.cs#L10-L11
I think it would be fine to use names without underscores in the unmanaged code under src/mono so that the style remains consistent.

Reply:
Ah, you are right, it works with not matching names, thanks!